diff --git a/config.def.h b/config.def.h
index b41747f..eae969e 100644
--- a/config.def.h
+++ b/config.def.h
@@ -5,9 +5,9 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char font[] = "Fixed:pixelsize=13:style=SemiCondensed";
-static int borderpx = 2;
-#define histsize 2000
+static char font[] = "Inconsolata:size=16:style=Medium:antialias=True:autohint=True;";
+static int borderpx = 8;
+#define histsize 8000
 
 /*
  * What program is execed by st depends of these precedence rules:
